Use Global Registration instead of  rpm (#642)
theforeman/foreman-documentation: guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc

.Procedure

To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, use one of the following procedures:

==== Global Registration
. Generate registration command
+
UI: Go to Hosts > Register Host, select activation key(s) and generate registration command.

> Or you can generate command via API:

Reviewer: I like the hammer example, but this says via the API. Could we additionally add an example below hammer for API like so:

```
curl https://username:password@$HOSTNAME/api/registration_commands -X POST -H 'Content-Type: application/json' -d '{\"activation_key\":\"your-key\",\"organization_id\":\"organization_id\"}'"
```

.Procedure

> To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, use one of the following procedures:

Reviewer 2: ```suggestion
To use an activation key for host registration with an existing {RHEL} 7 host to {ProjectServer}, use one of the following procedures:
```

==== Global Registration

> . Generate registration command

Reviewer 2: ```suggestion

. Generate registration command
```

+

> UI: Go to Hosts > Register Host, select activation key(s) and generate registration command.

Reviewer 2: ```suggestion
UI: In the {ProjectWebUI}, navigate to *Hosts > Register Host*, select one or more Activation Keys and generate the registration command.
```

Please replace the "UI" part and do not add "(s)" to pluralize a word: see IBM style guide "parentheses to form plurals".

Or you can generate command via API:
+
[options="nowrap" subs="+quotes,attributes"]
----
hammer host-registration generate-command --activation-keys your-key

> curl -X POST https://server.example.com/api/registration_commands \

Reviewer 2: Please add "#" or "$" prefix.

  --user username:password \
  -H 'Content-Type: application/json' -d '{\"activation_key\":\"your-key\",\"organization_id\":\"organization_id\"}'"
----

> +

Reviewer 2: ```suggestion
```

. Log on to the host you want register and run the command

==== Katello ca consumer rpm

> . Download the consumer RPM for your {ProjectServer}.

Reviewer 2: ```suggestion

. Download the consumer RPM for your {ProjectServer}.
```

To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, complete the following steps:

> . Download the consumer RPM for your {ProjectServer}.

Reviewer: I think we should keep this workflow but put it after the global registration one and mark it as deprecated.

> . On the host, enter the registration command & run it

Reviewer: Could we add a section below this on how to do this with the API?
